tools/install.sh

```diff
@@ -32,9 +32,9 @@ sed -i -e "/export PATH=/ c\\
 export PATH=\"$PATH\"
 " ~/.zshrc
 
-if [ "$SHELL" != "$(which zsh)" ]; then
+if [ "$SHELL" != "$(grep /zsh$ /etc/shells | tail -1)" ]; then
```

Comment:
I submitted [ptamarit/oh-my-zsh#1](https://github.com/ptamarit/oh-my-zsh/pull/1) to deal with that.